hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/HawkbitUI.java

```diff
@@ -180,7 +180,10 @@ public void afterViewChange(final ViewChangeEvent event) {
         final String locale = getLocaleId(SPUIDefinitions.getAvailableLocales());
         setLocale(new Locale(locale));
 
-        UI.getCurrent().setErrorHandler(new SPUIErrorHandler());
+        if (UI.getCurrent().getErrorHandler() == null) {
```

Comment:
Yes. It is necessary if you have a different UI class with a specific errorHandler. The HawkbitUI always overwrites the errorHandler then.